src/rpc/blockchain.cpp

```diff
@@ -509,17 +509,17 @@ static UniValue getmempoolancestors(const JSONRPCRequest& request)
 {
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
         throw std::runtime_error(
-            "getmempoolancestors txid (verbose)\n"
+            "getmempoolancestors txid ( verbose )\n"
```

Comment:
What is the original rationale behind the spacing in `( verbose )`?

Reply:
Unsure. Maybe to differentiate it from other normal options. It was also done with `( verbosity )`, so I matched it.